internal/test/bmc.go

```diff
@@ -0,0 +1,17 @@
+package test
```

Comment:
I can understand why this might make sense to go here, but if we are to ever get to a point where we can make our e2e test tool a completely different Go module, remove the spaghetti dependency tree, and drastically reduce the imports of our actual product code, eksctl-anywhere, we should not continue to add to this package tree. I recommend that we try to stick with the package tree that starts with the top level `test` directory/package. The `internal/test/cleanup` package would be ok for now but all that clean up code should ideally live in the top level `test` directory/package as well.

Also, i see that this function probably went here to that it can be consumed in 2 locations. Duplicating this code where it's used as opposed to creating a new dependency might be a good thing here. This is not a lot of code and taking on the extra code over the dependency, to me, makes sense. "[A little copying is better than a little dependency.](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=9m28s)" - https://go-proverbs.github.io/

Reply:
Thanks for calling this out.

Yeah, it went here so that it can be consumed in 2 locations. Now,  I agree with the principle that a little copying is better than a little dependency, especially since it's not a lot of code. I will go with duplicating this rather than introducing the dependency.